proto/osmosis/cosmwasmpool/v1beta1/model/v3/instantiate_msg.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+package osmosis.cosmwasmpool.v1beta1.model.v3;
+
+import "gogoproto/gogo.proto";
+import "osmosis/cosmwasmpool/v1beta1/model/v3/pool_query_msg.proto";
+
+option go_package = "github.com/osmosis-labs/osmosis/v25/x/cosmwasmpool/cosmwasm/msg/v3";
+
+// InstantiateMsg defines the message to instantiate a new alloyed asset
+// transmuter v3 pool.
+message InstantiateMsg {
+  repeated AssetConfig pool_asset_configs = 1 [ (gogoproto.nullable) = false ];
+  string alloyed_asset_subdenom = 2;
+  string alloyed_asset_normalization_factor = 3;
+  string admin = 4;
+  string moderator = 5;
+}
```

Comment:
non-blocking nit: consider defining it directly as json to remove the proto overhead.

Reply:
Sorry I need to clarify, do you mean add the json tags or rename these to something like alloyedAssetSubdenom

Reply:
I mean that instead of having it be a proto generated type, we can define a struct with json tags.

This is comment is non-blocking if you disagree

Reply:
Oh I see, was just following the last v1 implementation. Took your recommendation here https://github.com/osmosis-labs/osmosis/pull/8310/commits/a238932dd4022e65195e1ca5362af826163d1b61